### ISSUE TITLE
How to supply a value to selectRow column?

### ISSUE THREAD

**Issue by agushurst** [react-bootstrap-table/react-bootstrap-table2#953]
Is there a way to impart a value on the checkbox inputs when the selectRow attribute is added to a table? When I supply a selectRow attribute, I see the checkbox column appear, but I can't seem to find a way to add a value attribute to the checkbox inputs. For instance, I would like to express:

selectRow = {
        mode: "checkbox",
        clickToSelect: true,
        dataField: "id"
};

This would add a value attribute to each checkbox input, e.g., <input type="checkbox" value="..."> that contains the value coming from the "id" dataField. That way, I can easily get the selected values from the checkboxes. I do not want to display the "id" field in my table.

If there is a way to accomplish the above use case currently, I would appreciate any direction. Thanks much!

P.S. I tried creating my own checkbox column, but I can't find a way to replace the column header text with a checkbox (when I do that, I get an error as your control is expecting text rather than an object).

**Comment by AllenFang** [comment]
> Is there a way to impart a value on the checkbox inputs when the selectRow attribute is added to a table?

hello, you can use [`selectRow.selectionRenderer`](https://react-bootstrap-table.github.io/react-bootstrap-table2/docs/row-select-props.html#selectrowselectionrenderer-function)

**Comment by agushurst** [comment]
Thanks for the response. I found what I needed under the "Exposed API" in the Storybook. The example showed how to get the currently selected id's with the following sample code:

  handleGetSelectedData = () => {
    console.log(this.node.selectionContext.selected);
  }

Following this example, I was able to get the id's from the user selection. Closing the issue as the intent of my issue is already supported. Thanks!
